presto-spi/src/main/java/com/facebook/presto/spi/type/UnscaledDecimal128Arithmetic.java

```diff
@@ -127,6 +127,19 @@ public static Slice unscaledDecimal(BigInteger unscaledValue)
         return decimal;
     }
 
+    public static Slice unscaledDecimal(long low, long high, boolean negative)
+    {
+        Slice decimal = unscaledDecimal();
+        unscaledDecimal(low, high, negative, decimal);
+        return decimal;
+    }
+
+    public static void unscaledDecimal(long low, long high, boolean negative, Slice result)
```

Comment:
there is a `pack` method that does the same for `v1`, `v2`, `v3`, `v4`. I also have pack method with lo, hi in my casts PR. Make naming consistent. I would prefer `pack` because `unscaledDecimal` methods take other value types (e.g. double, long, BigInteger, etc). What do you think?